apps/src/StudioApp.js

```diff
@@ -582,29 +583,9 @@ StudioApp.prototype.init = function (config) {
     );
   }
 
+  this.initSettingsUI();
   this.initVersionHistoryUI(config);
 
-  if (this.isUsingBlockly() && Blockly.contractEditor) {
-    Blockly.contractEditor.registerTestsFailedOnCloseHandler(
```

Comment:
`registerTestsFailedOnCloseHandler` doesn't exist anywhere else in our repo. This is safe to delete.